src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -84,7 +85,17 @@ public SyntaxNode Mutate(SyntaxNode currentNode)
             }
             else if (currentNode is IfStatementSyntax ifStatement)
             {
-                return MutateIfStatement(ifStatement);
+                var mutateIfStatement = MutateIfStatement(ifStatement);
```

Comment:
I don't think we should change our mutating logic for this mutator. If we target expressions in the mutator, our current mutating logic will call the mutator at the right nodes for us.

Reply:
Actually it turned out there was no way of doing this without changing the orchestrator...